### ISSUE TITLE
Markers need to move with clip cuts

### ISSUE THREAD

**Issue by timbze** [jliljebl/flowblade#651]
Please include the following information.
1) Flowblade version (Help->About): 2.0.0
2) MLT version (Help->Runtime Environment): 6.12.0
3) Your distribution (Ubuntu, Debian, Mint etc.): Ubuntu 19.04
4) If possible, try to provide step-by-step instructions on how to reproduce the issue. From developer point of view this is the easieast way to understand what is happening.:

Scenario 1
1. I add a timeline marker at the 10 minute mark
2. I put the Mark in at 8:30, and Mark out at 9:00
3. I use the "Range Delete" tool
What happens: The timeline marker stays at 10:00
What I think should happen: The timeline marker moves to 9:30

Scenario 2
1. I put a clip marker on V1 at 10:00
2. I use cut tool at 9:30
What happens: The mark on the clip V1 at 10:00 disappears. If I right click the clip to the left of the cut, before 9:30, and look in Markers submenu, it still shows the 10:00 marker.
What I think should happen: The mark on clip V1 should be moved to the clip on the right of the cut, keeping it visible, and showing when I check the Markers submenu on the right hand side clip, not left.
Furthermore, if I cut a piece of clip V1 before the 10:00 clip marker, the clip marker should also move as far left as I have cut, but it stays at the same 10:00 location.

**Comment by jliljebl** [comment]
At first look I think Scenario 2 is something that should/could be fixed. 

Timeline markers are more transient and in my view simply mark a point in timeline and are not meant to react to edits because it is not clear which clip should be kept in sync when e.g. clips on track V1 move but clips on track V2 stay in the same position.

I will return to this after Python 3 conversion, leaving this without label for the time being.

**Comment by timbze** [comment]
Thanks for considering it!
If Scenario 2 was fixed I would be set for myself.
To try clarifying my point about Scenario 1. A "Range Delete" takes out the selected portion from all clips, essentially shortening the entire timeline. In my case on doing the Range Delete, I always would want those markers moved. But maybe in professional editing there are ways in which you simply want a marker at a certain time mark, I don't know.

On another note, I have used Lightworks quite a bit, and it only has the timeline markers, and they used to operate the same way Flowblade operates now. But in one of the last updates they changed it to operate in how I am asking now.
